packages/macro/src/macroJsx.test.ts

```diff
@@ -1,18 +1,17 @@
 import { parseExpression as _parseExpression } from "@babel/parser"
 import * as types from "@babel/types"
 import MacroJSX, { normalizeWhitespace } from "./macroJsx"
-import { JSXElement } from "@babel/types"
 
 const parseExpression = (expression: string) =>
   _parseExpression(expression, {
     plugins: ["jsx"],
-  }) as JSXElement
+  }) as any
 
 function createMacro() {
   return new MacroJSX({ types }, { stripNonEssentialProps: false })
 }
 
-describe("jsx macro", () => {
+xdescribe("jsx macro", () => {
```

Comment:
This unit tests now quite hard to fix. Because there no way to parse a codesnippet and receive a `path` as it was done with `node`. 

I will try to fix it, but if not, we can simply delete them because they're just testing the same things as e2e test (which is more preferable in this context)

Reply:
Fixed